src/main/scala/com/microsoft/hyperspace/index/IndexLogEntry.scala

```diff
@@ -111,6 +102,43 @@ object Content {
       None
     }
   }
+
+  /**
+   * Apply `func` to each file in directory recursively.
+   *
+   * @param prefixPath Root prefix
+   * @param directory Root directory
+   * @param func Function which would apply to current prefix and file
+   * @tparam T
+   * @return Result list of applying function to all files
+   */
+  def recFilesApply[T](
```

Comment:
```suggestion
  private[hyperspace] def recFilesApply[T](
```

btw is there any reason to move the function to here `object Content`?